OpenStack-Rabbit-Consumer/usr/local/bin/message_consumer.py

```diff
@@ -117,23 +118,27 @@ def consume(message):
             vmhost = message.get("payload").get("host")
             firstip = message.get("payload").get("fixed_ips")[0].get("address")
 
+            logger.error("Creating machine")
+
             try:
                 machinename = aq_api.create_machine(
                     uuid, vmhost, vcpus, memory_mb, hostname, prefix)
             except Exception as e:
-                raise Exception("Failed to create machine %s",e)
-                logger.error("Failed to create machine %s",e)
-
+                raise Exception("Failed to create machine {0}".format(e))
+                logger.error("Failed to create machine {0}".format(e))
```

Comment:
Raising before logging means you'll never log the error. Do you even need to re-raise it?